packages/react-geo/package.json

```diff
@@ -69,7 +69,7 @@
       "name": "MapView",
       "path": "dist/esm/index.mjs",
       "import": "{ MapView }",
-      "limit": "258 kB"
+      "limit": "259 kB"
```

Comment:
lol

Reply:
It changed by 51 bytes in JS lib, pushing us over 258